docs/getting-started-guides/binary_release.md

```diff
@@ -3,6 +3,8 @@
 
 You can either build a release from sources or download a pre-built release.  If you do not plan on developing Kubernetes itself, we suggest a pre-built release. 
 
+If you just want to run Kubernetes locally, we recommend using Minikube. You can download Minikube [here](https://github.com/kubernetes/minikube/releases/latest).
```

Comment:
"Just want to run locally" like what? For development, not in production, right? Without the use of a VM solution in non-Linux OSes? Little more clarity on what this does.